lldb/packages/Python/lldbsuite/test/lldbplatformutil.py

```diff
@@ -308,13 +308,21 @@ def expectedCompilerVersion(compiler_version):
         return operator in [">", ">=", "!", "!=", "not"]
 
     if operator == ">":
-        return LooseVersion(test_compiler_version) > LooseVersion(version)
+        return packaging.version.parse(test_compiler_version) > packaging.version.parse(
```

Comment:
all these would probably be easier to read if we factored out both parse calls into a variable, as they are repeated everywhere. Don't feel strongly about it though